### DISCUSSION TITLE
Possibility to include NCAA games

### DISCUSSION THREAD

**Discussion by ricomix23** [kyleskom/NBA-Machine-Learning-Sports-Betting#406] [Q&A] [answered]
Hello all,
Thank you for your great job. Do you think that it's possible to decline the project for NCAA games? In this case, what should be done? On my side I have in mind:
- Get teams data after each day of the season (is it right that model needs all those data and not only the current teams statistics?)
- Get odds for NCAA games
- Store those data into the database.

Thank you for your answers :-)

**Comment by nealmick** [comment, marked as answer]
Thats correct.  mainly just need to get and form data, a model can be applied to any dataset fairly easily.  Just keep in mind that no model is infallible and the models in this repo are not profitable by them self.   If your just looking for profitable sports betting strategies arbitrage and market consensus value betting are both good options.  No matter what strategy you use if you win a significant amount you will be limited by bookies and no longer be able to place bets.
